src/tye_lab_to_nwb/neurotensin_valence/images/neurotensinconfocalimagesdatainterface.py

```diff
@@ -0,0 +1,115 @@
+import re
+from datetime import datetime
+from typing import Optional
+
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.tools.nwb_helpers import make_or_load_nwbfile
+from neuroconv.utils import FilePathType, dict_deep_update
+from oiffile import OifFile
+from pynwb import NWBFile
+from pynwb.base import Images
+from pynwb.image import GrayscaleImage
+
+
+class NeurotensinConfocalImagesInterface(BaseDataInterface):
+    """Primary interface for converting confocal images for the Neurotensin experiment."""
+
+    def __init__(
+        self,
+        file_path: FilePathType,
+        verbose: bool = True,
+    ):
+        """
+        Interface for converting confocal images from Olympus FluoView.
+
+        Parameters
+        ----------
+        file_path : FilePathType
+            path to the Olympus Image File (.OIF).
+        verbose: bool, default: True
+            controls verbosity.
+        """
+        self.verbose = verbose
+        self.oif = OifFile(file_path)
+
+        super().__init__(file_path=file_path)
+
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+
+        metadata_from_file = dict()
+        settings = self.oif.filesystem.settings
+
+        # Add session_start_time
+        session_start_time = settings["Acquisition Parameters Common"]["ImageCaputreDate"]
```

Comment:
```suggestion
        session_start_time = settings["Acquisition Parameters Common"]["ImageCaptureDate"]
```

Reply:
Unless it is spelled that way in the settings?

Reply:
I had to check twice as well, but yes 😄 
<img width="562" alt="Screenshot 2023-03-27 at 11 39 51" src="https://user-images.githubusercontent.com/24475788/227904407-6ddeb7d2-05df-4d79-97ab-3ce8e2c25964.png">